src/docs/SUPPORTED_MODELS.md

```diff
@@ -178,6 +178,45 @@ The pipeline can work with other similar topologies produced by `optimum-intel`
   </tbody>
 </table>
 
+## Text 2 image models
+
+<table>
+  <tbody style="vertical-align: top;">
+    <tr>
+      <th>Architecture</th>
+      <th>Example HuggingFace Models</th>
+    </tr>
+    <tr>
+      <td><code>Stable Diffusion</code></td>
+      <td>
+        <ul>
+          <li><a href="https://huggingface.co/botp/stable-diffusion-v1-5"><code>botp/stable-diffusion-v1-5</code></a></li>
+          <li><a href="https://huggingface.co/stabilityai/stable-diffusion-2"><code>stabilityai/stable-diffusion-2</code></a></li>
+          <li><a href="https://huggingface.co/stabilityai/stable-diffusion-2-1"><code>stabilityai/stable-diffusion-2-1</code></a></li>
+          <li><a href="https://huggingface.co/dreamlike-art/dreamlike-anime-1.0"><code>dreamlike-art/dreamlike-anime-1.0</code></a></li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><code>Latent Consistency Model</code></td>
+      <td>
+        <ul>
+          <li><a href="https://huggingface.co/SimianLuo/LCM_Dreamshaper_v7"><code>SimianLuo/LCM_Dreamshaper_v7</code></a></li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><code>Stable Diffusion XL</code></td>
```

Comment:
Sorting the table alphabetically would be nice :) Maybe it's worth putting `Text 2 image models` before `Visual language models` for the same reason.

Reply:
done